ui/webpack.common.ts

```diff
@@ -120,6 +136,10 @@ module.exports = {
       API_PREFIX: API_BASE_PATH,
       STATIC_PREFIX: BASE_PATH,
     }),
+    new MonacoWebpackPlugin({
+      languages: ['json', 'javascript', 'go', 'markdown'],
```

Comment:
did we theme markdown and go? (read: implement the textmate grammer definition and color scheme)

Reply:
I don't believe so. I also think we should not have javascript and go now that I think about it.